src/components/tabs/AppTabs.vue

```diff
@@ -1,10 +1,16 @@
 <template>
-  <AppSelect v-model="selectedItem" :items="items" class="mb-4 md:hidden" />
-  <ul class="mb-4 hidden border-b border-primary-40 text-sm md:-mx-2 md:flex">
+  <AppSelect
+    v-model="selectedItem"
+    :items="items"
+    class="mb-4 md:mb-6 md:hidden"
```

Comment:
I don't think you need `md:mb-6` as `md:hidden` will hide the element in any case,

Reply:
Good catch; I missed that bit but it's now gone.